src/schemas/baks/begrunnelse/ba-sak/sanityMappeFelt/periodeType.tsx

```diff
@@ -0,0 +1,36 @@
+import { BegrunnelseDokumentNavn, Menyvalg, SanityTyper } from '../../../../../util/typer';
+import { Rule } from 'sanity';
+
+export enum PeriodeType {
+  UTBETALING = 'UTBETALING',
+  INGEN_UTBETALING = 'INGEN_UTBETALING',
+  INGEN_UTBETALING_UTEN_PERIODER = 'INGEN_UTBETALING_UTEN_PERIODER',
+  FORTSATT_INNVILGET = 'FORTSATT_INNVILGET',
+}
+
+export const periodeTypeTilMenyValg = (periodeType: PeriodeType): Menyvalg<PeriodeType> => {
+  const periodeTypeTilMenynavn = (periodeType: PeriodeType): string => {
+    switch (periodeType) {
+      case PeriodeType.UTBETALING:
+        return 'Utbetaling';
+      case PeriodeType.INGEN_UTBETALING:
+        return 'Ingen utbetaling';
+      case PeriodeType.INGEN_UTBETALING_UTEN_PERIODER:
+        return 'Ingen utbetaling uten perioder';
+      case PeriodeType.FORTSATT_INNVILGET:
+        return 'Fortsatt innvilget';
+    }
+  };
+
+  return { title: periodeTypeTilMenynavn(periodeType), value: periodeType };
+};
+
+export const periodeType = {
+  title: 'Resultat',
+  type: SanityTyper.STRING,
+  name: BegrunnelseDokumentNavn.VEDTAK_RESULTAT,
```

Comment:
Skal det opprettes ny BegrunnelseDokumentNavn her for periodeType?

Reply:
[Fiks alle steder vedtakresultat ikke var byttet ut med periodetype](https://github.com/navikt/familie-sanity-brev/pull/444/commits/787e5cf553071cd3fea92a1ce72c3d86f35b9af2)